styles.css

```diff
@@ -1,102 +1,198 @@
 * { 
-    box-sizing: border-box;
-  }
-
-  html {
-    background-image: linear-gradient(to bottom, rgba(48, 19, 117, 0.651), rgba(202, 180, 198, 0.651));
-  }
-  
-  body {
-    font-family: Helvetica, Arial, sans-serif;
-    font-size: 110%;
-    line-height: 1.5;
-    margin: 2em;
-    color:rgb(20, 15, 48);
-  }
-
-  h1, h2, h3, h4, h5, h6 {
-    line-height: 1.2;
-    text-decoration: underline;
-    text-align: center;
-  }
-
-  h1 {
-    font-family: Bradley Hand, Helvetica, Arial, sans-serif;
-  }
-
-  hr{
-    border: 0;
-    width: auto;
-    height: 1px;
-    background-color: rgba(72, 66, 83, 0.459);
-  }
-
-  a {
-    text-decoration: none;
-    text-align: center;
-    border-bottom: 1px solid rgba(56, 20, 77, 0.5);
-  }
-
-  small {
-    font-size: 1rem;
-  }
-
-  .main-nav {
-      border: 1px solid none;
-      height: 50px;
-      padding: 4px;
-      display: flex;
-      gap: 50px;
-      justify-content: flex-start;
-      align-items: flex-start;
-      align-content: flex-start;
-  }
-
-  .main-nav > * {
-    flex-grow: 1;
-    flex-basis: 0%;
-  }
-
-  .button {
-    background-color: rgba(96, 62, 136, 0.822);
-    padding: 8px 12px;
-    border-radius: 4px;
-    color: rgb(201, 170, 226);
-    box-shadow: 5px 4px 6px rgba(29, 3, 53, 0.8); 
-  }
-
-  .coverimg {
-    border-radius: 30px 190px 80px 190px;
-    box-shadow: 0px 0px 30px 1px;
-    display: flex;
-    margin-left: auto;
-    margin-right: auto;
-    width: auto;
-  }
-
-  .centre {
-    text-align: center;
-    margin-left: auto;
-    margin-right: auto;
-    width: 100%;
-  }
-
-  .buttontext {
-    background-color: rgba(159, 121, 190, 0.7);
-    padding: 1px;
-    border-radius: 4px;
+  box-sizing: border-box;
+}
+
+html {
+  background-image: linear-gradient(to bottom, rgba(48, 19, 117, 0.651), rgba(202, 180, 198, 0.651));
+}
+
+body {
+  font-family: Helvetica, Arial, sans-serif;
+  font-size: 110%;
+  line-height: 1.5;
+  margin: 1.5em;
+}
+
+h1 {
+  font-family: "Bradley Hand", Helvetica, Arial, sans-serif;
```

Comment:
just a question - as I see you didnt link that font in your html head - is "Bradley Hand" naturally on your pc? i will look after this review on mine 😃, this font name makes me curious, haha

Reply:
haha it seems to be on my pc. 😄
but what do you mean with linking in the head?